unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ChatWidgetHUD/ChatEntry.cs

```diff
@@ -126,6 +137,16 @@ public void Populate(Model chatEntryModel)
             body.text = $"{chatEntryModel.bodyText}";
         }
 
+        if (CoordinateUtils.HasValidTextCoordinates(body.text)) {
+            CoordinateUtils.GetTextCoordinates(body.text).ForEach(c=> {
+                int x = Int32.Parse(c.Split(',')[0]), y = Int32.Parse(c.Split(',')[1]);
+                if (MinimapMetadata.GetMetadata().GetSceneInfo(x,y) == null)
```

Comment:
perhaps this could be extracted into a method: `PreloadSceneMetadata(ParcelCoordinates coordinate)`